tests/mock_client.py

```diff
@@ -74,8 +74,8 @@ def invoke_list_rows(self, data: dict):
             ]
             rows += [
                 ListRowsResponse(
-                    hid=f"row-placeholder-{idx}",
-                    id=f"row-placeholder-{idx}",
+                    hid=f"row-{idx}",
+                    id=f"row-{idx}",
```

Comment:
having two `-` was breaking the logic in get_dataframe that sorted by indices. we know that indices will always have just one `-`, so this construction matches that